### PR TITLE
MM-24847 MM-24484 MM-24850 MM-24849 - local mode for commands

### PR DESCRIPTION
**Pull request by reflog**
#### Summary
This PR includes local mode support for update/delete/move/get operations on commands.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-24847
https://mattermost.atlassian.net/browse/MM-24850
https://mattermost.atlassian.net/browse/MM-24849
https://mattermost.atlassian.net/browse/MM-24848

### CODE REVIEW

**Comment by reflog** [comment]
@mgdelacroix can we get a review? pretty please? :)

File: api4/command_test.go [hunk 1]
```diff
@@ -107,51 +106,51 @@ func TestUpdateCommand(t *testing.T) {
 		Token:     "tokenchange",
 	}
 
-	rcmd, resp := Client.UpdateCommand(cmd2)
-	CheckNoError(t, resp)
+	th.TestForSystemAdminAndLocal(t, func(t *testing.T, client *model.Client4) {
```
Comment (mgdelacroix) [inline api4/command_test.go:109]: As these tests where originally running for `th.Client`, shouldn't this be:

```suggestion
	th.TestForAllClients(t, func(t *testing.T, client *model.Client4) {
```

File: api4/command_test.go [hunk 2]
```diff
@@ -171,23 +170,24 @@ func TestMoveCommand(t *testing.T) {
 	}
 
 	rcmd1, _ := th.App.CreateCommand(cmd1)
+	th.TestForSystemAdminAndLocal(t, func(t *testing.T, client *model.Client4) {
```
Comment (mgdelacroix) [inline api4/command_test.go:173]: Same suggestion here:

```suggestion
	th.TestForAllClients(t, func(t *testing.T, client *model.Client4) {
```

File: api4/command_test.go [hunk 3]
```diff
@@ -227,24 +226,26 @@ func TestDeleteCommand(t *testing.T) {
 		Trigger:   "trigger1",
 	}
 
-	rcmd1, _ := th.App.CreateCommand(cmd1)
-
-	ok, resp := Client.DeleteCommand(rcmd1.Id)
-	CheckNoError(t, resp)
-
-	require.True(t, ok)
+	th.TestForSystemAdminAndLocal(t, func(t *testing.T, client *model.Client4) {
```
Comment (mgdelacroix) [inline api4/command_test.go:229]: Same:

```suggestion
	th.TestForAllClients(t, func(t *testing.T, client *model.Client4) {
```

File: api4/command_test.go (hunk 1 above)
Comment (reflog) [reply api4/command_test.go:109]: @mgdelacroix  original client was th.SystemAdminClient so this should be TestForSystemAdminAndLocal

File: api4/command_test.go (hunk 2 above)
Comment (reflog) [reply api4/command_test.go:173]: @mgdelacroix  original client was th.SystemAdminClient so this should be TestForSystemAdminAndLocal

File: api4/command_test.go (hunk 3 above)
Comment (reflog) [reply api4/command_test.go:229]: same

File: api4/command_test.go (hunk 1 above)
Comment (mgdelacroix) [reply api4/command_test.go:109]: You're totally right! :facepalm:

**Pull request merged**